test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNet/BulkOperation/BulkOperationTest.cs

```diff
@@ -0,0 +1,172 @@
+            string untypedFriendsNavigationLink = "UnTypedFriends@odata.navigationLink";
+
+            //Act & Assert
+            using (HttpResponseMessage response = await this.Client.SendAsync(requestForPatch))
+            {
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                var json = response.Content.ReadAsStringAsync().Result;
+                Assert.Equal(expectedResponse.ToString().ToLower(), json.ToString().ToLower());
+                Assert.Contains("SqlUD", json);
+                Assert.Contains(friendsNavigationLink, json);
+                Assert.Contains(newFriendsNavigationLink, json);
+                Assert.Contains(untypedFriendsNavigationLink, json);
+            }
+        }
+
+        [Fact]
+        public async Task PostEmployee_WithFullMetadata()
+        {
+            //Arrange
+            string requestUri = this.BaseAddress + "/convention/Employees?$format=application/json;odata.metadata=full";
+
+            var content = @"{
+                    'Name':'SqlUD'
+                     }";
+
+            string expectedResponse = "{" +
+                "\"@odata.context\":\"" + this.BaseAddress + "/convention/$metadata#Employees/$entity\"," +
+                "\"@odata.type\":\"#Microsoft.Test.E2E.AspNet.OData.BulkOperation.Employee\"," +
+                "\"@odata.id\":\"" + this.BaseAddress + "/convention/Employees(0)\"," +
+                "\"@odata.editLink\":\"" + this.BaseAddress + "/convention/Employees(0)\",\"ID\":0,\"Name\":\"SqlUD\",\"SkillSet@odata.type\":\"#Collection(Microsoft.Test.E2E.AspNet.OData.BulkOperation.Skill)\",\"SkillSet\":[],\"Gender@odata.type\":\"#Microsoft.Test.E2E.AspNet.OData.BulkOperation.Gender\",\"Gender\":\"0\",\"AccessLevel@odata.type\":\"#Microsoft.Test.E2E.AspNet.OData.BulkOperation.AccessLevel\",\"AccessLevel\":\"0\",\"FavoriteSports\":null," +
+                "\"Friends@odata.associationLink\":\"" + this.BaseAddress + "/convention/Employees(0)/Friends/$ref\",\"Friends@odata.navigationLink\":\"" + this.BaseAddress + "/convention/Employees(0)/Friends\",\"NewFriends@odata.associationLink\":\"" + this.BaseAddress + "/convention/Employees(0)/NewFriends/$ref\",\"NewFriends@odata.navigationLink\":\"" + this.BaseAddress + "/convention/Employees(0)/NewFriends\",\"UnTypedFriends@odata.associationLink\":\"" + this.BaseAddress + "/convention/Employees(0)/UnTypedFriends/$ref\"," +
+                "\"UnTypedFriends@odata.navigationLink\":\"" + this.BaseAddress + "/convention/Employees(0)/UnTypedFriends\",\"#Microsoft.Test.E2E.AspNet.OData.BulkOperation.AddSkill\":{\"title\":\"AddSkill\",\"target\":\"" + this.BaseAddress + "/convention/Employees(0)/Microsoft.Test.E2E.AspNet.OData.BulkOperation.AddSkill\"},\"#Microsoft.Test.E2E.AspNet.OData.BulkOperation.GetAccessLevel\":{\"title\":\"GetAccessLevel\",\"target\":\"" + this.BaseAddress + "/convention/Employees(0)/Microsoft.Test.E2E.AspNet.OData.BulkOperation.GetAccessLevel()\"}}";
```

Comment:
What are these? Are they the actions/functions of the entity type?

Reply:
Yes they are operations.